src/components/Input/index.tsx

```diff
@@ -0,0 +1,13 @@
+import * as Styled from "./style";
+
+import { IInput } from "./types"
+
+export default function Input({ labelContent, inputPlaceHolder }:IInput) {
```

Comment:
conforme o que sugeri na interface, você pode alterar este para:

```suggestion
export default function Input({ labelContent, ...props }:IInput) {
```